modules/ROOT/pages/logging/log-forwarding.adoc

```diff
@@ -1,24 +1,27 @@
 [[aura-query-logs]]
 = Security log forwarding
 
-label:AuraDB-Enterprise[]
+label:AuraDB-Business-Critical[]
+label:AuraDB-Virtual-Dedicated-Cloud[]
 label:AuraDS-Enterprise[]
 
 With security log forwarding, you can stream security logs directly to a cloud project owned by your organization, in real time.
 
 To access *Log forwarding*:
 
-. Navigate to the https://console.neo4j.io/[Neo4j Aura console] in your browser.
-. Select *Log forwarding* from the sidebar menu.
+. Navigate to *Settings* under the Project section in the sidebar.
+. Click *Log forwarding*. If you cannot see this option, you may not have the necessary permissions.
```

Comment:
```suggestion
. Select *Log forwarding*. 
If you cannot see this option, you may not have the necessary permissions.
```
Is a certain role required to see this option?

Reply:
Yes, Project Admin is required. Perhaps I should explicitly mention that.